js/ui/tooltips/color-picker.js

```diff
@@ -101,11 +101,18 @@ TooltipEngine.classes.colorPicker = TooltipBase.extend({
         };
         this.aceLocation.tooltipCursor = this.aceLocation.start + this.aceLocation.length + this.closing.length;
 
-        if (event.source && event.source.action === "insertText" && 
+        var name = event.line.substring(functionStart, paramsStart - 1);
+        var addSemicolon =
+            this.isAfterAssignment(event.pre.slice(0, functionStart));
+        if (['fill', 'stroke'].includes(name)) {
```

Comment:
We include `browser-polyfill.min.js` in `live-editor.shared.js`.  Also, `includes` is already in use in other places of the code.